new_platforms/vsextension/ProjectWizard/WizardImplementation.cs

```diff
@@ -83,6 +83,13 @@ private bool SetOETADevKitPath(Dictionary<string, string> replacementsDictionary
             string unixPath = "/mnt/" + drive + "/" + relativeFolder;
 
             replacementsDictionary.Add("$OETADevKitPath$", unixPath);
+
+            string buildFlavor = board.Split('\\')[0];
+            replacementsDictionary.Add("$OpteeBuildFlavor$", buildFlavor);
+
+            string opteeCompilerFlavor = (buildFlavor == "vexpress-qemu_virt") ? "arm-linux-gnueabihf-" : "aarch64-linux-gnu-";
```

Comment:
This is also an issue if users pick their own dev kit. We can't assume which compiler it's correct for their board.

Reply:
true, but not a regression, at least by default we have a valid answer here (aarch64 being the default).  Will cover as part of future changelist for "Other" boards